Bucketeer2/Sources/Internal/Utils/Constant.swift

```diff
@@ -0,0 +1,19 @@
+import Foundation
+
+struct Constant {
+    static let MINIMUM_FLUSH_INTERVAL_MILLIS: Int64 = 60_000 // 60 seconds
+    static let DEFAULT_FLUSH_INTERVAL_MILLIS: Int64 = 60_000 // 60 seconds
+    static let DEFAULT_MAX_QUEUE_SIZE: Int = 50
+    static let MINIMUM_POLLING_INTERVAL_MILLIS: Int64 = 60_000 // 60 seconds
+    static let DEFAULT_POLLING_INTERVAL_MILLIS: Int64 = 600_000 // 10 minutes
+    static let MINIMUM_BACKGROUND_POLLING_INTERVAL_MILLIS: Int64 = 1_200_000 // 20 minutes
+    static let DEFAULT_BACKGROUND_POLLING_INTERVAL_MILLIS: Int64 = 3_600_000 // 1 hour
+
+    struct DB {
+        static let FILE_NAME = "bucketeer.db"
+        static let VERSION: Int32 = 2
+    }
+
+    static let RETRY_POLLING_INTERVAL: Int64 = 1_000 * 60 // 1 minute
```

Comment:
Perhaps we can define as you did in the `MINIMUM_POLLING_INTERVAL_MILLIS`

Eg.
```swift
 60_000 // 60 seconds
 ```

Reply:
Changed https://github.com/ca-dp/bucketeer-ios-sdk/commit/144e2a02474e5861ada9727fafb82a4683956c4c#diff-40e43c403c7c358119ce8379696fd48de49c2aab2de0ef6be7c329bc51671477L17-R17